scalafix/rules/src/main/scala/org/typelevel/fix/v0_5.scala

```diff
@@ -0,0 +1,14 @@
+package org.typelevel.fix
+
+import scalafix.v1._
+import scala.meta._
+
+class v0_5 extends SemanticRule("v0_5") {
+  override def fix(implicit doc: SemanticDocument): Patch =
+    doc.tree.collect {
+      case Importer(ref , _) if ref.toString == "io.github.davidgregory084" =>
```

Comment:
Unlikely, but you don't have anything else under this namespace that could get caught up in this right? 😁